src/gz/explorer.c

```diff
@@ -316,20 +310,26 @@ static int draw_proc(struct menu_item *item,
       gsSPSegment(0x0B, &null_dl),
       gsSPSegment(0x0C, &null_dl),
       gsSPSegment(0x0D, &null_dl),
+      gsSPTexture(0xFFFF, 0xFFFF, 0, G_TX_RENDERTILE, G_OFF),
       gsSPLoadGeometryMode(G_ZBUFFER | G_SHADE | G_CULL_BACK | G_LIGHTING |
                            G_SHADING_SMOOTH),
-      /* rdp settings */
-      gsDPSetAlphaCompare(G_AC_NONE),
+      /* clear Z buffer */
+      gsDPPipeSync(),
+      gsDPSetOtherMode(G_AD_DISABLE | G_CD_DISABLE | G_CK_NONE | G_TC_FILT |
+                       G_TF_BILERP | G_TT_NONE | G_TL_TILE | G_TD_CLAMP |
+                       G_TP_NONE | G_CYC_1CYCLE | G_PM_NPRIMITIVE,
+                       G_AC_NONE | G_ZS_PRIM | CVG_DST_CLAMP | FORCE_BL |
+                       ZMODE_OPA | Z_UPD | IM_RD |
+                       GBL_c1(G_BL_CLR_IN, G_BL_A_IN, G_BL_CLR_MEM, G_BL_1MA) |
+                       GBL_c2(G_BL_CLR_IN, G_BL_A_IN, G_BL_CLR_MEM, G_BL_1MA)),
+      gsDPSetCombineLERP(0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0),
+      gsDPSetPrimDepth(0x7FFF, 0),
+      gsDPFillRectangle(0, 0, Z64_SCREEN_WIDTH - 1, Z64_SCREEN_HEIGHT - 1),
```

Comment:
Minor complaint, I think this should be `Z64_SCREEN_WIDTH` and `Z64_SCREEN_HEIGHT` (sans the `-1`) when the DP is set to 1-cycle. @Thar0 do you agree?

Reply:
Seems right to me based on the docs so I've made this change

Reply:
Late but yeah this is right, nice catch 😌